code/__DEFINES/jobs.dm

```diff
@@ -22,7 +22,6 @@
 #define CMO_JF				(1<<3)
 #define DOCTOR			(1<<4)
 #define GENETICIST		(1<<5)
-#define VIROLOGIST		(1<<6)

```

Comment:
Since you are removing the virologist can you add one doctor role to balance out how many there are for the department? (At times the virologist basically acts as a normal doctor)

Reply:
@KorPhaeron this pls

Reply:
Giving people a chance to make alternative PRs before I put more work into this one but I will